build.gradle

```diff
@@ -502,6 +511,13 @@ project(':aeron-cluster') {
     }
 
     task generateCodecs(type: JavaExec) {
+        def codecsFile = 'src/main/resources/aeron-cluster-codecs.xml'
+        def markCodecsFile = 'src/main/resources/aeron-archive-mark-codecs.xml'
```

Comment:
Should this not be `aeron-cluster-mark-codecs.xml`

Reply:
Yes, you are right. Thanks for spotting this.
Fixed